code/modules/paperwork/fax.dm

```diff
@@ -537,8 +537,10 @@ GLOBAL_VAR_INIT(nt_fax_department, pick("NT HR Department", "NT Legal Department
 		target_fax.receive(fax_item, sender)
 
 	else if(force) //no fax machines but we really gotte send? SEND A FAX MACHINE
-		var/obj/machinery/fax/new_fax_machine = new ()
-		send_supply_pod_to_area(new_fax_machine, area_type, force_pod_type)
+		var/obj/machinery/fax/new_fax_machine = new()
+		if(!send_supply_pod_to_area(new_fax_machine, area_type, force_pod_type))
+			log_mapping("Attempted to forcibly send a fax to [area_type], however the area does not exist or has no valid dropoff spot for a fax machine")
```

Comment:
this shouldn't log to `log_mapping()` because no one checks log_mapping except when it fails during unit tests. please replace this with a `stack_trace()` instead as that is infinitely more trackable.

Reply:
Is there a better solution to this? Like I said in the PR description, having the debugger break on a debug map like runtimestation is something I'd like to avoid

Reply:
the event that specifically targets virology should check if virology exists before sending a fax to it and this should still stack trace. it's making the assumption that every map has a virology department, which is not a correct assumption that should be made as three maps on the repository do not have a virology department.